force-app/main/default/classes/FeatureParameters.cls

```diff
@@ -189,14 +189,17 @@ public without sharing class FeatureParameters {
 
     @TestVisible
     private without sharing class ActiveProgramsWithServiceDeliveriesLast30 implements FeatureManagement.FeatureParameter {
-        private string programField =
-            ServiceDelivery__c.ProgramEngagement__c.getDescribe().getRelationshipName() +
-            '.' +
-            String.valueOf(ProgramEngagement__c.Program__c);
+        private String childRelationshipName = Util.getChildRelationshipName(
+            ProgramEngagement__c.SObjectType,
+            ServiceDelivery__c.SObjectType
+        );
+        private string programField = String.valueOf(ProgramEngagement__c.Program__c);
+        private QueryBuilder subQueryBuilder = new QueryBuilder()
```

Comment:
Don't we have to use this subQueryBuilder in a query or am I confused?

Reply:
Oops, nice catch. That was part of the original plan that didn't pan out. I'll remove and push shortly.